resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+              <td>
+                <p>lisaManagerReferenceNumber is in the wrong format</p>
+                <p class="code--block">
+                  lisaManagerReferenceNumber: 123456<br>
+                  accountId: 1234567890<br>
+                  lifeEventId: 1234567891
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+          {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+          }
+        </pre>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                  <p>accountId is in the wrong format</p>
+                  <p class="code--block">
+                    lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>
+                    accountId: 1234%3D5678<br>
+                    lifeEventId: 1234567891
+                  </p>
+                </td>
+                <td>
+                  <p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter accountId in the correct format, like ABC12345"
+          }
+        </pre>
+                </td>
+            </tr>
+            <tr>
+              <td>
+                <p>Account not found</p>
+                <p class="code--block">
+                  lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>
+                  accountId: 0000000404<br>
+                  lifeEventId: 1234567891
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p>Life event not found</p>
+                <p class="code--block">
+                  lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br>
+                  accountId: 1234567890<br>
+                  lifeEventId: 0000000404
+                </p>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "LIFE_EVENT_NOT_FOUND",
+          "message": "The lifeEventId does not match with HMRC’s records"
+        }
+        </pre>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      operationId: viewLifeEvent
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
```

Comment:
This does not require a Content-Type header (the RAML is incorrect) as this is a retrieval endpoint.

<img width="659" alt="Screenshot 2023-02-10 at 09 04 41" src="https://user-images.githubusercontent.com/63714619/218049688-945cfa47-fde0-46a9-a894-6ddf23ea9410.png">

Please correct this at least in this PR by removing this reference. Thanks.

<img width="960" alt="Screenshot 2023-02-10 at 12 20 34" src="https://user-images.githubusercontent.com/63714619/218104004-0345dd6f-a4bd-4151-9ebd-f6cc89da8cf8.png">